packages/user-interface/src/styles/_definitions.scss

```diff
@@ -2,4 +2,4 @@ $breakpoints: (
   mobile: 320px,
   tablet: 768px,
   desktop: 1024px,
-);
+) !default;
```

Comment:
Wat doet dit?

Reply:
Dit zorgt er voor dat de breakpoints variabele op andere plekken niet overschreven kan worden, en als deze variabele eerder al wel gedefinieerd is wordt deze niet overschreven. (https://github.com/stylelint-scss/stylelint-scss/blob/master/src/rules/dollar-variable-default/README.md)